netbeans/CheckerFrameworkPlugin/src/org/checkerframework/netbeans/CheckerFrameworkPanel.java

```diff
@@ -0,0 +1,139 @@
+package org.checkerframework.netbeans;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import javax.swing.BoxLayout;
+import javax.swing.JCheckBox;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import org.netbeans.api.project.ProjectManager;
+import org.netbeans.spi.project.support.ant.EditableProperties;
+import org.openide.filesystems.FileObject;
+import org.openide.filesystems.FileUtil;
+import org.openide.modules.InstalledFileLocator;
+import org.openide.util.MutexException;
+
+public class CheckerFrameworkPanel extends JPanel {
+
+    private EditableProperties editableProperty;
+    private static Map<String, String> checkerStrings;
+    private final FileObject projectProperties;
+    private final JLabel title;
```

Comment:
Do you even need this field? It looks like it's only used in the constructor and then never again.
A one-line comment for each field would also be good.